packages/phoenix-event-display/src/tests/managers/three-manager/scene-manager.test.ts

```diff
@@ -0,0 +1,235 @@
+/**
+ * @jest-environment jsdom
+ */
+import { SceneManager } from '../../../managers/three-manager/scene-manager';
+import { Object3D, Scene, Vector3 } from 'three';
+import {
+  AmbientLight,
+  AxesHelper,
+  BoxGeometry,
+  BufferGeometry,
+  Camera,
+  Color,
+  DirectionalLight,
+  Group,
+  Mesh,
+  MeshBasicMaterial,
+} from 'three';
+import { CoordinateHelper } from '../../../helpers/coordinate-helper';
+
+describe('SceneManager', () => {
+  let sceneManager: SceneManager;
+  const ignoreList = [
+    new AmbientLight().type,
+    new DirectionalLight().type,
+    new AxesHelper().type,
+  ];
+
+  beforeEach(() => {
+    sceneManager = new SceneManager(ignoreList, false);
+  });
+
+  it('should update position of directional light for each frame rendered', () => {
+    sceneManager.updateLights(new Camera());
+    expect(sceneManager.cameraLight).toBeUndefined();
```

Comment:
This does not test the functionality correctly. We need to make sure that the `cameraLight` has the position of `camera`.